third_party/externals.bzl

```diff
@@ -346,6 +354,14 @@ def register_sorbet_dependencies():
         strip_prefix = "shellcheck-v{}".format(shellcheck_version),
     )
 
+    # Needed to build CMake projects
+    http_archive(
+        name = "rules_foreign_cc",
+        urls = _github_public_urls("bazelbuild/rules_foreign_cc/archive/refs/heads/main.zip"),
+        sha256 = "c1ec32b37bf802357356c5fc63f1de927ad080bffca4899af56816adf1d366e5",
+        strip_prefix = "rules_foreign_cc-main",
```

Comment:
Let's get the commit hash of `main` instead of putting `main` here. Using `main` will mean that when a commit gets pushed to rules_foreign_cc, anyone who didn't already have rules_foreign_cc in their build cache will get a checksum failure.